backend/app.ts

```diff
@@ -20,9 +20,13 @@ app.use(express.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "client/build")));
 
 // -------------------routes
-// app.use("/company", companyRouter);
+app.get("/", (request: Request, response: Response) => {
+  // console.log(request.query);
```

Comment:
We may want logging infrastructure at some point, but until then let's get rid of extraneous debug logs

Reply:
Yup. I'll make sure to delete the unused comments or codes before I merge them.